dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+# This script is expected to be run with a PWD of the root of the puppet-server
+# project.
+
+
+# Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+# Legacy means pre-aio (4.0) in this case
+export PUPPET_LEGACY_VERSION='3.7.5'
+
+  # --post-suite acceptance/suites/post_suite_backwards_compatibility \
+bundle exec beaker \
+  --debug \
+  --root-keys \
+  --no-color \
+  --repo-proxy \
+  --preserve-hosts never \
+  --type aio \
+  --config acceptance/config/beaker/jenkins/redhat7-64m-64a.yaml \
```

Comment:
The static configuration is necessary so long as the `genconfig2` command isn't normally available.  By "normal" I mean expressed as a non-special-case bundler dependency via the Gemfile.

@waynr I'm happy to switch this script over to dynamically creating the beaker configuration, but for that to happen we need to figure out a way eliminate the `GEM_SOURCE` special case behavior.  How do you recommend we eliminate this special-case? https://github.com/puppetlabs/pe-razor-server/blob/37ae577f789dc2e49101b9ce4d3b97b0c20efd20/acceptance/scripts/testrun_install-from-package.sh#L16-L17